src/test/java/com/kaltura/client/test/tests/servicesTests/AssetHistoryTests/AssetHistoryActionCleanLists.java

```diff
@@ -28,13 +29,17 @@
 
     @BeforeClass
     private void add_tests_before_class() {
-        client = getClient(getSharedMasterUserKs());
+
 
     }
 
     @Description("/assetHistory/action/clean - no filtering")
     @Test
     private void cleanHistory() {
+
+        Household household = HouseholdUtils.createHouseHold(1, 1, false);
```

Comment:
please use variables to avoid "magic numbers" like int numberOfUsers=1; int numberOfDevices=1; HouseholdUtils.createHouseHold(numberOfUsers, numberOfDevices, false);